src/chrome/content/rules/Ibisworld.com.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Ibisworld.com">
+	<target host="ibisworld.com" />
+	<rule from="^http://ibisworld.com" to="https://www.ibisworld.com" />
+
```

Comment:
@gloomy-ghost: Unfortunately, as far as I can see, this would not be quite sufficient in fixing the issue, as the faulty reference is not due to a HTTPS Everywhere and will persist even with an exclusion. One way around it would be to add a downgrade rule for that particular URL, changing HTTPS requests to HTTP (but be aware that such downgrades come with problems of their own; in particular, if HTTPS will be forced by t he site on that URL at some point in the future, we end up in a situation, where we are *removing* security rather than adding it).